android/KMAPro/kMAPro/src/main/java/com/tavultesoft/kmapro/MainActivity.java

```diff
@@ -479,38 +476,48 @@ public void onKeyboardDismissed() {
   /**
    * Parse the URI data to determine the filename and URL for the .kmp keyboard package.
    * If URL is valid, download the kmp.
-   * @param scheme String of the URI's scheme.
```

Comment:
Note that the `scheme` parameter was never used: replacing it with the URI for the download makes much more sense 🤣

Reply:
Thanks for catching and cleaning this. I *think* at one point it was to install from keyman:// scheme